src/modules/tests/test-data/cat-be/manoeuvres/manoeuvres.cat-be.selectors.ts

```diff
@@ -0,0 +1,6 @@
+import { CatBEUniqueTypes } from '@dvsa/mes-test-schema/categories/BE';
+import { get } from 'lodash';
+
+export const getReverseLeftSelected = (manoeuvres: CatBEUniqueTypes.Manoeuvres) => {
```

Comment:
should this be isReverseLeftSelected

Reply:
Could rename it, although it's getting the Reverse Left Selected property so it's okay to call it a get